Consul.AspNetCore/Consul.AspNetCore.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <Version>1.0.0</Version>
```

Comment:
It would make sense to have Consul.AspNetCore's version match Consul's version IMO.

Reply:
There is no need for that. If `Consul.AspNetCore` is not changed in any way then there is no need to publish its new version. These two libraries have independent release cycle so there is no need for both of them to use the same versioning scheme.

Reply:
It's a matter of taste, I guess. Having dependent packages share their version numbers and release cycle makes it dead simple for the user to choose their packages, and avoids any kind of nuget reference problems. Several multi-packages libs follow this strategy (SkiaSharp, libvlcsharp, etc.).

No strong feelings though.